core/templates/dev/head/components/forms/FormBuilderSpec.ts

```diff
@@ -182,6 +193,11 @@ describe('Testing requireIsFloat directive', function() {
       '<input name="floatValue" type="number" ng-model="localValue" ' +
       'require-is-float apply-validation>' +
       '</form>';
+    scope.validators = function() {
```

Comment:
The tests failed because `scope.validators` was undefined, I don't know why the tests passed before, but it makes sense to have this defined because it is done the same in the other tests.

Reply:
This is needed for the tests to work, I don't know why they worked previously without this, but this is set in the other test cases too.